tests/unit/test_standalone.py

```diff
@@ -53,13 +53,18 @@ def test_standalone_problem(self):
 
         # Test the Problem with a Cost
         rmse_cost = pybop.RootMeanSquaredError(problem)
-        x = rmse_cost([1, 2])
+        rmse_x = rmse_cost([1, 2])
+        rmse_grad_x = rmse_cost.evaluateS1([1, 2])
 
-        np.testing.assert_allclose(x, 3.138, atol=1e-2)
+        np.testing.assert_allclose(rmse_x, 3.05615, atol=1e-2)
```

Comment:
This change was a bit worrisome, since the value should remain constant. I did a comparison with Pints' logic and it returned the updated value as well, pointing towards a bug in the previous RMSE calculation.